test/mocha/cli.js

```diff
@@ -94,6 +94,35 @@ describe("bin/uglifyjs", function () {
             done();
         });
     });
+    it("Should not load source map before finish reading from STDIN", function(done) {
+        var mapFile = "tmp/input.js.map";
+        try {
+            fs.mkdirSync("./tmp");
+        } catch (e) {
+            if (e.code != "EEXIST") throw e;
+        }
+        try {
+            fs.unlinkSync(mapFile);
+        } catch (e) {
+            if (e.code != "ENOENT") throw e;
+        }
+        var command = [
+            uglifyjscmd,
+            "--source-map", "content=" + mapFile + '"',
```

Comment:
Looks like this trailing quote slipped back in somehow... :sweat_smile:

Reply:
I've pushed the fix to your PR, so no further actions required :crossed_fingers: